app/assets/javascripts/discourse/app/lib/notification-types/liked.js

```diff
@@ -4,12 +4,13 @@ import { i18n } from "discourse-i18n";
 export default class extends NotificationTypeBase {
   get label() {
     const nameOrUsername = this.siteSettings.prioritize_full_name_in_ux
-      ? this.notification.data.display_name
+      ? this.notification.data.display_name ||
+        this.notification.data.display_username
```

Comment:
though, wouldn't it be shorter/easier/more performant/etc to have the notification include only one (as `display_name`) based on the site setting?

Reply:
Talked it over, and ultimately I think for now this is going to be the least intrusive way forward, given that it's behind a setting. Absolutely room in the future for a refactor though, so I'm adding that to my refactor list. 👍